lib/logr.rb

```diff
@@ -0,0 +1,45 @@
+require "logger"
+
+require "logr/version"
+require "logr/entry"
+require "logr/json_formatter"
+
+module Logr
+  class Logger
+    def initialize(name, level=::Logger::DEBUG, log_device=STDOUT)
```

Comment:
Shouldn't `INFO` be the default? I would think that's the accepted default. What does this value control exactly?